docs/import.md

```diff
@@ -1,74 +1,101 @@
-Importing Source Files in Solidity
+Importing & Loading Source Files in Solidity
 ==================================
 
-There are multiple techniques for importing files into Remix.
+There are two main reasons for loading external files into Remix:
+- **to import a library or dependancy** (for files you will NOT be editing)
+- **to load some files for manipulation, editing and play** (for files you might want to edit)
 
-For a tutorial about importing files click [here](https://github.com/ethereum/remix-workshops/tree/master/LoadingContent).  You can also find this tutorial in the Remix Workshops plugin.
+## Importing a library or dependancy
 
-For a detailed explanation of the `import` keyword see the
-[Solidity documentation](https://solidity.readthedocs.io/en/develop/layout-of-source-files.html?highlight=import#importing-other-source-files)
+When importing from NPM, or a URL (like github, a IPFS gateway, or a Swarm gateway) you do not need to do anything more than use the `import` statement in your contract. The dependencies do not need to be "preloaded" into the File Explorer's current Workspace before the contract is compiled.
 
-Here are a some of the main methods of importing a file:
+Files loaded from the import statement are placed in the **Files Explorer's** current Workspace's `.deps` folder. 
 
-Importing a file from the browser's local storage
--------------------------------------------------
+Under the hood, Remix checks to see if the files are already loaded in the **.deps** directory.  If not, it goes out and gets them from a url - if provided or via unpkg (if it is an NPM lib).
```

Comment:
`get them from the url`

Reply:
`via unpkg (if it is an NPM lib)` : Rather than putting `if it is an NPM lib`  in brackets, we should make it a statement for explanation to users

Reply:
don't we need an example here?